lib/extensions/src/lib/components/viewer/preview-extension.component.ts

```diff
@@ -43,6 +57,10 @@ export class PreviewExtensionComponent implements OnInit, OnChanges, OnDestroy {
     @Input()
     extension: string;
 
+    @Output()
+    contentLoaded = new EventEmitter<void>();
+
+    private readonly destroyRef = inject(DestroyRef);
```

Comment:
please put empty line between private and private readonly

Reply:
Done